### PR TITLE
Add r-intamap

### PR DESCRIPTION
**Pull request by daf**
ping @ocefpaf 

Needs:
- [X] r-automap (conda-forge/staged-recipes#3414)
- [X] r-evd (conda-forge/staged-recipes#3413)
- [X] gdal/r-rgdal issue fixed (conda-forge/r-rgdal-feedstock#2, https://github.com/conda-forge/r-rgdal-feedstock/pull/3 or https://github.com/conda-forge/r-rgdal-feedstock/pull/4)

### CODE REVIEW

File: recipes/r-intamap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - r-sp >=0.9
+
+test:
+  commands:
+    - $R -e "library('intamap')"  # [not win]
+    - "\"%R%\" -e \"library('intamap')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=intamap
+  license: GPL (>= 2)
```
Comment (scopatz) [inline recipes/r-intamap/meta.yaml]: GPL requires that the license file be distributed with the package

Comment (scopatz) [reply recipes/r-intamap/meta.yaml]: So please include a `license_file` entry

**Comment by scopatz** [comment]
Thanks for putting in this PR @daf!

**Comment by daf** [comment]
Thanks @scopatz, added!

**Comment by scopatz** [comment]
Thanks! This looks good to go to me!

**Pull request merged**